### DISCUSSION TITLE
Does Security Onion upload any of our data outside our location?

### DISCUSSION THREAD

**Discussion by thetayo** [Security-Onion-Solutions/securityonion#7507] [General] [answered]
We're looking to deploy SecurityOnion in our environment as a SIEM monitoring tool. I have reviewed this with my team and the major concern we have is whether SecutityOnion uploads any of our data into the cloud and what exact data do they upload but I can't seem to find any resources online that explicitly state that. I was wondering if anyone here has any ideas/knowledge on this?

**Comment by dougburks** [comment, marked as answer]
If you're doing an on-prem deployment, then all data stays on the Security Onion nodes in your environment. The only way that Security Onion would upload your data to the cloud would be if you deployed your manager and search nodes in the cloud and had your forward nodes sending there.
